File: beware-of-the-sheep/src/Entities/Animals/CyberSheep.cpp
#include "pch.h"
#include "CyberSheep.h"

namespace bots {

	CyberSheep::CyberSheep(World & world, const Point & position) :
		Animal(world, position, 11, 4)
	{}

	void CyberSheep::action()
	{
	}

	void CyberSheep::collision(Entity & other)
	{
		Animal::collision(other);
	}

} // namespace bots

File: src/com/kostu96/bots/entities/animals/CyberSheep.java
package com.kostu96.bots.entities.animals;

import com.kostu96.bots.World;
import com.kostu96.bots.entities.Entity;
import com.kostu96.bots.utils.Point;

public class CyberSheep extends Animal {

	public CyberSheep(World world, Point position, int strength, int initiative) {
		super(world, position, strength, initiative);
	}

	@Override
	public void collision(Entity other) {

	}

}
